osh.spec

```diff
@@ -219,6 +220,7 @@ fi
 %{_unitdir}/osh-stats.*
 %exclude %{python3_sitelib}/osh/hub/settings_local.py*
 %exclude %{python3_sitelib}/osh/hub/__pycache__/settings_local.*
+%dir %attr(775,root,root) /var/log/osh
```

Comment:
Why do we need to create it on the hub?

Reply:
Otherwise the `/var/log/osh` would be unowned if we installed `osh-hub` only.